### PR TITLE
update swarm to v1.1.0

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <vieux@docker.com>

### CODE REVIEW

**Comment by vieux** [comment]
ping @dongluochen

**Comment by dongluochen** [comment]
LGTM

File: library/swarm [hunk 1]
```diff
@@ -1,10 +1,9 @@
-# maintainer: Andrea Luzzardi <al@docker.com> (@aluzzardi)
-# maintainer: Victor Vieux <vieux@docker.com> (@vieux)
 # maintainer: Alexandre Beslic <abronan@docker.com> (@abronan)
+# maintainer: Andrea Luzzardi <al@docker.com> (@aluzzardi)
 # maintainer: Dong Chen<dong@docker.com> (@dongluochen)
-
-1.1.0-rc3: git://github.com/docker/swarm-library-image@73403dbc9a5ebfa2f48c331abe815c0228dd94b4
+# maintainer: Victor Vieux <vieux@docker.com> (@vieux)
```
Comment (tianon) [inline library/swarm]: Oh man, you know my crazy well; `:sort` :heart: :heart:

**Comment by tianon** [comment]
LGTM

Build test of #1416; 739e168 (`swarm`):

```console
$ bashbrew build "swarm"
Cloning swarm (git://github.com/docker/swarm-library-image) ...
Processing swarm:1.0.0 ...
Processing swarm:1.0.1 ...
Processing swarm:1.1.0 ...
Processing swarm:latest ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing swarm:1.0.0
	'cve-2014--shellshock' [1/1]...passed
testing swarm:1.0.1
	'cve-2014--shellshock' [1/1]...passed
testing swarm:1.1.0
	'cve-2014--shellshock' [1/1]...passed
```

**Pull request merged**